instructure/canvas-ios
# Make Google Cloud Assignments more consistent

refs: MBL-13810
affects: student
release note: Fixed inconsistent Google Cloud Assignment behavior

Test plan:
* Log in to narmstrong.instructure.com : s (preferably on an iPad)
* CS 1400 > Assignments > Scroll down a bit to Google Cloud Assignment
* Tap Launch External Tool
* The launch should not happen in SFSafari but instead in a full screen web view
* Tap Authorize then log in with a Google account (i don't have a test account for this, i just used my own)
* After you authorize the authentication modal should dismiss and you should get the full g doc editor
* Tap Done and launch again. You should go straight to the document
